db/dbtest/util.go

```diff
@@ -0,0 +1,71 @@
+package dbtest
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	dbm "github.com/cosmos/cosmos-sdk/db"
+)
+
+func AssertValid(t *testing.T, itr dbm.Iterator, expected bool) {
+	valid := itr.Valid()
+	require.Equal(t, expected, valid)
+}
+
+func AssertNext(t *testing.T, itr dbm.Iterator, expected bool) {
```

Comment:
`RequireNext`?

Reply:
I am pretty indifferent to the naming, but already renamed these once - would `Require` be the overall preference?

Reply:
Both are fine. We are using more `Assert*` in non test files.